scripts/test/Muon/fitting_widgets/basic_fitting/basic_fitting_presenter_test.py

```diff
@@ -430,6 +462,12 @@ def _setup_mock_view(self):
         type(self.view).start_x = self.mock_view_start_x
         self.mock_view_end_x = mock.PropertyMock(return_value=self.end_x)
         type(self.view).end_x = self.mock_view_end_x
+        self.mock_view_exclude_range = mock.PropertyMock(return_value=True)
```

Comment:
are we getting to a point where it might be worth putting these setup methods in a different file?

Reply:
These property mocks aren't easily moved to a different file because they are part of the test class. Normal mock methods have been moved to https://github.com/mantidproject/mantid/blob/4e3edf60bb556bdf1de046309270711e1ce1fc33/scripts/Muon/GUI/Common/test_helpers/fitting_mock_setup.py#L1

Reply:
I just think the file is getting a bit long and it would be better if we could split things up/make it easier to navigate